src/css/base/_variables_coa.scss

```diff
@@ -26,13 +63,33 @@ $coa-textwrap-width: 545px;
   justify-content: space-between;
 }
 
-// COA COLORS
+//colors
```

Comment:
did these come from the Sketch "sticker-sheet" page?

Reply:
I was thinking it might be nice to create a storybook story called typography to reflect the 'sticker-sheet' in Sketch

Reply:
Yep, they came from her stickersheet -- with some clarification since there were a couple errors. 
I'm down to create a storybook for the stickersheet. Honestly I haven't really been paying attention storybooks atm in favor of just finishing the service page for testing.